packages/medusa/src/api/routes/store/collections/index.ts

```diff
@@ -22,11 +22,12 @@ export default (app) => {
   return app
 }
 
-export const defaultStoreCollectionRelations = ["products"]
+export const defaultStoreCollectionRelations = []
 export const allowedFields = [
   "id",
   "title",
   "handle",
+  "product",
```

Comment:
```suggestion
  "products",
```

Reply:
😅  Fixed in 99451633aa5d39a82a02f5115a94beafb58425a3